hedera-node/src/test/java/com/hedera/services/txns/consensus/MerkleTopicUpdateTransitionLogicTest.java

```diff
@@ -1,49 +1,34 @@
 package com.hedera.services.txns.consensus;
 
-/*-
```

Comment:
nit: License was removed

Reply:
I also think we should **rename** this test to `TopicUpdateTransitionLogicTest` - in order to match the actual subject we're testing.